lang/syn/src/parser/accounts/constraints.rs

```diff
@@ -182,6 +182,15 @@ pub fn parse_token(stream: ParseStream) -> ParseResult<ConstraintToken> {
             };
             ConstraintToken::Bump(Context::new(ident.span(), ConstraintTokenBump { bump }))
         }
+        "program_seed" => {
```

Comment:
Is there any place in the existing solana sdk that uses the term "program_seed"?

Reply:
Tag @jstarry. Any ideas for an accurate keyword to refer to the base program id for PDAs?

Reply:
Yeah tbh I also find that term a little odd

Reply:
other words from the discord: `seeds::program`, `program`, `derive_from`, `seed_program`

Reply:
Discussed offline. Let's do `seeds::program`.